### PR TITLE
Upgraded to latest JRE 8u121

### PR DESCRIPTION
**Pull request by susanxhuynh**
Upgraded to latest JRE.

### CODE REVIEW

File: sdk/common/src/main/java/com/mesosphere/sdk/offer/Constants.java [hunk 1]
```diff
@@ -19,7 +19,7 @@
     public static final String EXECUTOR_URI = "EXECUTOR_URI";
     public static final String LIBMESOS_URI = "LIBMESOS_URI";
     public static final String JAVA_URI = "JAVA_URI";
-    public static final String DEFAULT_JAVA_URI = "https://downloads.mesosphere.com/java/jre-8u112-linux-x64.tar.gz";
+    public static final String DEFAULT_JAVA_URI = "https://downloads.mesosphere.com/java/jre-8u121-linux-x64.tar.gz";
```
Comment (loren) [inline sdk/common/src/main/java/com/mesosphere/sdk/offer/Constants.java]: this constant appears to be unused in the codebase

Comment (susanxhuynh) [inline sdk/common/src/main/java/com/mesosphere/sdk/offer/Constants.java]: I'll remove it then.

**Comment by susanxhuynh** [comment]
Thanks, @loren I'll update the ones inside the frameworks as well.

**Comment by nickbp** [comment]
Has this been done?

**Comment by susanxhuynh** [comment]
@nickbp I forgot to merge this PR. I'll rebase and try again.

**Comment by gabrielhartmann** [comment]
@susanxhuynh:  This PR is old, please re-open if this is still needed.

**Pull request closed without merging**